Sources/Paywalls/Components/Common/ComponentOverrides.swift

```diff
@@ -33,13 +33,25 @@ public extension PaywallComponent {
 
     }
 
-    enum Condition: String, Codable, Sendable, Hashable, Equatable {
+    enum Condition: Codable, Sendable, Hashable, Equatable {
 
-        case compact
-        case medium
-        case expanded
```

Comment:
Shouldn't we map these three (compact, medium and expanded) to the new `screenSize` ?